Assets/Scripts/Manager/GhostSwitchManager.cs

```diff
@@ -116,6 +116,10 @@ public void updateDolls ()
 		}
 	}
 
+	void updateAudio(){
+		MechanicAudioManager.getInstance ().updateAudioControllers ();
```

Comment:
I didn't put in update cos it's a particular case of updating audio upon ghost switch, it doesn't happen all that often so I don't put it in update
and I use the manager to keep track on the audio sources that needs to be updated upon ghostSwitch, it's a list in audio manager
I also thought would be helpful to keep track of mechanic audio sources in a manager